compiler/rustc_hir_typeck/src/method/suggest.rs

```diff
@@ -3228,6 +3229,61 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         }
     }
 
+    fn suggest_bounds_for_range_to_method(
+        &self,
+        err: &mut Diag<'_>,
+        source: SelfSource<'tcx>,
+        item_ident: Ident,
+    ) {
+        if let SelfSource::MethodCall(rcvr_expr) = source
+            && let hir::ExprKind::Struct(qpath, _, _) = rcvr_expr.kind
+            && let hir::QPath::LangItem(lang_item, _) = qpath
+        {
+            let is_inclusive = match lang_item {
+                hir::LangItem::RangeTo => false,
+                hir::LangItem::RangeToInclusive => true,
```

Comment:
```suggestion
                hir::LangItem::RangeToInclusive | hir::LangItem::RangeInclusiveCopy => true,
```

cc https://github.com/rust-lang/rust/issues/125687